src/incubator/TextField/Label.tsx

```diff
@@ -28,6 +28,7 @@ const Label = ({
   const style = useMemo(() => {
     return [styles.label, labelStyle, floatingPlaceholder && styles.dummyPlaceholder];
   }, [labelStyle, floatingPlaceholder]);
+  const shouldRenderIdication = context.isMandatory && showMandatoryIndication;
```

Comment:
Notice you have a typo here (shouldRenderIdication -> shouldRenderIndication)
Best way to spot typos is to install VSCode extension `Code Spell Checker`